### DISCUSSION TITLE
Parsing be default when adding recipes

### DISCUSSION THREAD

**Discussion by Coax88** [mealie-recipes/mealie#2913] [Ideas]
Hi, I find it cumbersome that when you import recipes from a URL, you have to go through the process of parsing the ingredients. Why can't this be automatically handled during the import, with the option to edit or correct it later?

**Comment by RealFoxie** [comment]
Sometimes you might not want it parsed and just use the original string. If you auto parse, it will show that parsed data, which could be wrong.
automatically parsing would force onto the user to instantly fix it.

Besides that, automatic parsing doesn't always work well for non-English languages, so it isn't something everyone would want to use.

It could be an option to do at import though, but it can't be the default for these reasons.

**Reply by Coax88** [reply]
I understand your point, but yes what I meant was having an option to make it happen automatically, just like you have the option to disable ingredient amounts. If you utilize the shopping list feature, using parsed recipes becomes essential. otherwise, it can become quite cluttered.

**Reply by ovizii** [reply]
And news on this feature suggestion by any chance?

**Reply by michael-genson** [reply]
Nightly (and the next release) has a brand-new parsing workflow which isn't _quite_ fully automatic parsing, but comes quite close.
More info here: https://github.com/mealie-recipes/mealie/pull/6151
